lib/handlers/error-pages.js

```diff
@@ -9,12 +9,36 @@ function handler (err, req, res, next) {
 
   var ldp = req.app.locals.ldp
 
-  if (err.status === 401 &&
-      req.accepts('text/html') &&
-      ldp.auth === 'oidc') {
-    debug('401 error - redirect to Select Provider')
-    res.status(err.status)
-    redirectToLogin(req, res, next)
+  if (ldp.auth === 'oidc') {
```

Comment:
Is there a way of including this in the OIDC code rather than in the global error handler?

`WWW-Authenticate` is meant to be used for all 401s, so it should be here for WebID-TLS as well.